pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -206,7 +207,9 @@ func TestCRIListPodStats(t *testing.T) {
 	checkCRIRootfsStats(assert, c1, containerStats1, nil)
 	checkCRILogsStats(assert, c1, &rootFsInfo, containerLogStats1)
 	checkCRINetworkStats(assert, p0.Network, infos[sandbox0.PodSandboxStatus.Id].Stats[0].Network)
-	checkCRIPodCPUAndMemoryStats(assert, p0, infos[sandbox0Cgroup].Stats[0])
+	if runtime.GOOS == "linux" {
```

Comment:
Hmm, usually we prefer to do this at compile time, but since this is just a test I guess it's fine.